GetJobProfiles/Models/Recipe/Fields/Factories/ContentPickerContentItemFactory.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using GetJobProfiles.Models.Recipe.ContentItems.Base;
+
+namespace GetJobProfiles.Models.Recipe.Fields.Factories
+{
+    public class ContentPickerContentItemFactory<T> where T : ContentItem
+    {
+        private readonly EqualityComparer<T> _comparer;
+        public readonly ConcurrentDictionary<T, string> ItemToUsers;
```

Comment:
Is "ItemToUsers" a typo? Might not be I'm just confused

Reply:
Hmm yeah, it's a bit confusing. I was thinking of 'users of the entry route'. Let me think of a better name.